package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "extract": "formatjs extract 'pages/**/*.js' 'components/**/*.jsx' --out-file 'locales/en.json' --format 'locales/locale_format.js'"
```

Comment:
gotcha so it would pull in the unneeded description by default? we can omit it then. i'd just add a code comment on the `locale_format.js` file to say what it's for / why it's different than the default handler